Yosemite/Yosemite/PointOfSale/POSProduct.swift

```diff
@@ -1,11 +1,11 @@
 
-public struct POSProduct: POSItem {
+struct POSProduct: POSItem {
     public let itemID: UUID
     public let productID: Int64
     public let name: String
     public let price: String
```

Comment:
super nit: these can probably be internal now
```suggestion
    let itemID: UUID
    let productID: Int64
    let name: String
    let price: String
```

Reply:
The can! https://github.com/woocommerce/woocommerce-ios/pull/12931/commits/cf0b935d0eb492774a047bc094a71840a2c80d57